Build/installation/usage

Hello,
could you _PLEASE_ make a better tutorial on that... I'm not really familiar with neither linux nor cmake nor git nor anything.

When cloning from master, I get
```
test cases:  1 |  1 failed
assertions: 12 | 12 failed
```

[pastebin](https://pastebin.com/wChpWLDa)

When I downloaded the last release (v1.2.0), I had a different error, and it didn't even build
```
[ 20%] Building CXX object test/CMakeFiles/catchmain.dir/main.cpp.o
In file included from /usr/include/signal.h:328,
                 from /media/keychain/test/catch.hpp:7937,
                 from /media/keychain/test/main.cpp:7:
/media/keychain/test/catch.hpp:10717:58: error: call to non-‘constexpr’ function ‘long int sysconf(int)’
10717 |     static constexpr std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
      |                                                          ^~~~~~~~~~~
In file included from /usr/include/x86_64-linux-gnu/bits/sigstksz.h:24,
                 from /usr/include/signal.h:328,
                 from /media/keychain/test/catch.hpp:7937,
                 from /media/keychain/test/main.cpp:7:
/usr/include/unistd.h:640:17: note: ‘long int sysconf(int)’ declared here
  640 | extern long int sysconf (int __name) __THROW;
      |                 ^~~~~~~
In file included from /media/keychain/test/main.cpp:7:
/media/keychain/test/catch.hpp:10776:45: error: size of array ‘altStackMem’ is not an integral constant-expression
10776 |     char FatalConditionHandler::altStackMem[sigStackSize] = {};
      |                                             ^~~~~~~~~~~~
gmake[3]: *** [test/CMakeFiles/catchmain.dir/build.make:76: test/CMakeFiles/catchmain.dir/main.cpp.o] Error 1
gmake[2]: *** [CMakeFiles/Makefile2:129: test/CMakeFiles/catchmain.dir/all] Error 2
gmake[1]: *** [CMakeFiles/Makefile2:189: test/CMakeFiles/test.dir/rule] Error 2
gmake: *** [Makefile:208: test] Error 2
-- Install configuration: ""
CMake Error at _build/cmake_install.cmake:46 (file):
  file INSTALL cannot find "/media/keychain/_build/libkeychain.a": No such
  file or directory.
```

However, I tried it with the files created by 1st method, but it didn't work. I added `keychain` to the linker and no luck.
![image](https://user-images.githubusercontent.com/7432818/187297651-6b832d7f-9286-4353-9b27-9f5d389699ce.png)

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -47,9 +47,8 @@ jobs:
       - name: Build and run tests
         run: |
           if [ "$RUNNER_OS" == "Linux" ]; then
-            export DISPLAY=:99.0
-            eval $(dbus-launch --sh-syntax)
-            echo '$(whoami)' | gnome-keyring-daemon -r -d --unlock
+            eval $(DISPLAY=:99.0 dbus-launch --sh-syntax)
+            echo "somepassword" | gnome-keyring-daemon -r -d --unlock
           fi
           cmake --build . --target test
 

